packages/frontend/src/components/accounts/SetupSeedPhraseVerify.js

```diff
@@ -72,66 +72,74 @@ const SetupSeedPhraseVerify = (
         reset() {
             debugLog('in imperative handle reset()');
             return recaptchaRef.current.reset();
-        }
+        },
     }));
 
-    const shouldRenderRecaptcha = !ENABLE_IDENTITY_VERIFIED_ACCOUNT && !isLinkDrop && CONFIG.RECAPTCHA_CHALLENGE_API_KEY && isNewAccount;
+    const shouldRenderRecaptcha =
+        !ENABLE_IDENTITY_VERIFIED_ACCOUNT &&
+        !isLinkDrop &&
+        CONFIG.RECAPTCHA_CHALLENGE_API_KEY &&
+        isNewAccount;
 
     return (
         <CustomDiv>
             <h4 data-test-id="seedPhraseVerificationWordNumber">
-                <SafeTranslate
-                    id="input.enterWord.title"
-                    data={{ wordId: wordId + 1 }}
-                />
+                <SafeTranslate id="input.enterWord.title" data={{ wordId: wordId + 1 }} />
             </h4>
             <Translate>
                 {({ translate }) => (
                     <input
                         data-test-id="seedPhraseVerificationWordInput"
-                        name='enterWord'
+                        name="enterWord"
```

Comment:
I've noticed a few double-quote symbols in this file. ESLint used to give us linting errors about that, but not it seems it doesn't care.

To be consistent with the old repo, we should keep the old rules and way they styled things. So please try get the linting working to ensure single-quotes are used for strings.

Prettier is still set correctly, so maybe we also just need to run that on a commit hook. But also don't want too many files to change all of a sudden now though- so the lint warning should suffice.